lib/src/fake_aggregate_query.dart

```diff
@@ -28,10 +31,18 @@ class FakeAggregateQuery implements AggregateQuery {
   @override
   AggregateQuery count() => _query.count();
 
+  Map<String, dynamic> _getRawDocDataMap(QueryDocumentSnapshot<Object?> s) {
+    if (s is MockQueryDocumentSnapshot && s.snapshot is MockDocumentSnapshot) {
```

Comment:
Why not use an assert statement like you did in the other PR? `flutter test` does fail if the assert fails. If something does fails for unexpected reasons, let it fail so we can fix it properly next time. :)

Reply:
Oh, it's because I want to check the type with the `is` operator so that we can use the instance method without casting it again.
Let me think and revise it. ;)

Reply:
after using `DocumentSnapshot.get` we don't need this method anymore, haha. Thank you!